lightblock.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"github.com/btcsuite/btcd/blockchain"
+	"github.com/btcsuite/btcd/wire"
+)
+
+type LightBlock struct {
+	height  int32
+	header  *wire.BlockHeader
```

Comment:
why making a pointer? This struct is small.

Reply:
Tbh, my question is "It's working!"  :') .

Reply:
let's use non pointer - for small data structures it's safer and faster.

Reply:
Updated